pkg/types/clusterconfig/cluster_config.go

```diff
@@ -112,6 +113,7 @@ type ManagedConfig struct {
 	OperatorLoadBalancerCIDRWhiteList []string           `json:"operator_load_balancer_cidr_white_list,omitempty" yaml:"operator_load_balancer_cidr_white_list,omitempty"`
 	VPCCIDR                           *string            `json:"vpc_cidr,omitempty" yaml:"vpc_cidr,omitempty"`
 	CortexPolicyARN                   string             `json:"cortex_policy_arn" yaml:"cortex_policy_arn"` // this field is not user facing
+	AccountID                         string             `json:"account_id" yaml:"account_id"`               // this field is not user facing
```

Comment:
Do you think it would make sense to save the hashed account ID instead of (or in addition to) the account ID?

Reply:
Hashing the account ID should be deterministic, right?